src/risedevtool/redis.toml

```diff
@@ -0,0 +1,26 @@
+extend = "common.toml"
+
+[env]
+REDIS_VERSION = "7.0.4"
+REDIS_DOWNLOAD_PATH = "${PREFIX_TMP}/redis-${REDIS_VERSION}.tar.gz"
+REDIS_RELEASE = "redis-${REDIS_VERSION}"
+REDIS_DOWNLOAD_URL = "https://github.com/redis/redis/archive/${REDIS_VERSION}.tar.gz"
+
+[tasks.download-redis]
+category = "RiseDev - Components"
+dependencies = ["prepare"]
+condition = { env_set = [ "ENABLE_REDIS" ] }
+description = "Download and extract Redis"
+script = '''
+#!/bin/bash
+set -e
+if [ -d "${PREFIX_BIN}/redis" ]; then
+    exit 0
+fi
+echo "Kafka not found, download ${REDIS_RELEASE}"
+curl -fL -o "${REDIS_DOWNLOAD_PATH}" "${REDIS_DOWNLOAD_URL}"
+tar -xvzf "${REDIS_DOWNLOAD_PATH}" -C "${PREFIX_TMP}"
+mv "${PREFIX_TMP}/${REDIS_RELEASE}" "${PREFIX_BIN}/redis"
+rm ${REDIS_DOWNLOAD_PATH}
+cd "${PREFIX_BIN}/redis" && make -j all
```

Comment:
Don’t redis have pre-built binaries? Looks unacceptable for me to compile it from source.

Reply:
I am also surprised but it does not have one. The compile takes 10+s on my machine and looks like it is acceptable.

Reply:
Does it need any dependency to compile?

Reply:
No